bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -404,6 +448,10 @@
 		<item-type>String</item-type>
 		<label>Water Level</label>
 		<description>Shows the water level of a WeMo enabled Holmes Humidifier</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Mode</tag>
```

Comment:
Not sure what is correct, but `Mode` seems wrong.

Reply:
Changed to `Level`